docs/StardustDocs/topics/gettingStarted.md

```diff
@@ -1,22 +1,170 @@
-[//]: # (title: Getting started)
+[//]: # (title: Get started with DataFrame)
 
-The Kotlin DataFrame library is a JVM Kotlin library for in-memory data manipulation.
+Kotlin's DataFrame library gives you the power to manipulate data in your Kotlin projects. 
 
-![getting started image](gettingStarted.png)
+This page explains how to:
+* Set up the DataFrame library in an IntelliJ IDEA project with Gradle.
+* Import and manipulate data.
+* Export data.
 
-## Installation
-First, pick up the latest version of the Kotlin DataFrame library [here](https://search.maven.org/artifact/org.jetbrains.kotlinx/dataframe).
-* If you wish to play with data in interactive mode, setup [Kotlin Kernel for Jupyter](installation.md#jupyter-notebook) and run DataFrame there
-* If you have some JVM project, just add a dependency on the Kotlin DataFrame library like it's described on [Maven Central search](https://search.maven.org/artifact/org.jetbrains.kotlinx/dataframe) site
+To use the DataFrame library with Jupyter notebooks or DataLore, follow the instructions on our [Installation page](installation.md).
 
-## Examples
-Hope that this documentation will help you to implement all the things you want to do with your data. To get inspiration, take a look at [examples](https://github.com/Kotlin/dataframe/tree/master/examples) folder. [Puzzles](https://github.com/Kotlin/dataframe/blob/master/examples/jupyter-notebooks/puzzles/40%20puzzles.ipynb) will quickly familiarize you with the power of DataFrame, and other notebooks and projects will show you some applications of DataFrame in practical data analysis.
+## Install Kotlin
 
-## Feedback and contributing
-If you found a bug, or have an idea for a new feature, please [file an issue](https://github.com/Kotlin/dataframe/issues/new) in DataFrame GitHub repository.
+Kotlin is included in each IntelliJ IDEA release.
+Download and install [IntelliJ IDEA](https://www.jetbrains.com/idea/download/) to start using Kotlin.
 
-If you wish to contribute, you're welcome! Choose an issue you like, let us know that you're working on it, and prepare a [pull request](https://github.com/Kotlin/dataframe/pulls). We'll review and merge it if everything goes well.
+## Create Kotlin project
 
-You can also give us feedback or ask a question in [#datascience](https://kotlinlang.slack.com/archives/C4W52CFEZ) channel of a Kotlin Slack.
+1. In IntelliJ IDEA, select **File** | **New** | **Project**.
+2. In the panel on the left, select **New Project**.
+3. Name the new project and change its location, if necessary.
 
-Good luck!
+   > Select the **Create Git repository** checkbox to place the new project under version control. You can enable this
+   > later at any time.
+   >
+   {type="tip"}
+
+4. From the **Language** list, select **Kotlin**.
+5. Select the **Gradle** build system.
+6. From the **JDK list**, select the [JDK](https://www.oracle.com/java/technologies/downloads/) that you want to use in
```

Comment:
Probably we need to refer to the list of appropriate JVMs or just directly write (at least Java 8)

Reply:
Updated! If you're happy that this is resolved, please resolve this conversation.